Monika After Story/game/script-affection.rpy

```diff
@@ -170,30 +180,26 @@ init python:
     #Internal cooldown to avoid topic spam and Monika affection swings, the amount of time to wait before a function is effective
     #is equal to the amount of points it's added or removed in minutes.
 
-    #Monika's initial affection based on start-up. Need to decide on super exp before we do anything else...
-    #Monika closed game herself and how happy she is determines on time between closed game and reopening.
-    #if persistent.closed_self == True:
-       # if datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(hours = 6):
-
-
-       # elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(hours = 6) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(hours = 12):
-
-
-       # elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(hours = 12) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(hours = 18):
-
-
-        #elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(hours = 18) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 1):
-
-
-        #elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(days = 1) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 2):)
-
-
-       # elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(days = 2) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 4):
-
-
-        #elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(days = 4) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 14):
-
-
+    #Monika's initial affection based on start-up.
+    if persistent._mas_long_absence == False:
+        if persistent.sessions["last_session_end"] == None:
+            pass
+        else:
+            time_difference = datetime.datetime.now() - persistent.sessions["last_session_end"]
+            if time_difference >= datetime.timedelta(hours = 6) and time_difference <= datetime.timedelta(hours = 12):
```

Comment:
You don't have to change this part, but for future ref, this is valid python syntax:
```python
if <lower bound> <= <value to check> <= <higher_bound>:
```
So use that if you're checking a range (if it's easier for you to understand it that way).